packages/react-native/scripts/ios-configure-glog.sh

```diff
@@ -42,7 +42,7 @@ EOF
     patch -p1 config.sub fix_glog_0.3.5_apple_silicon.patch
 fi
 
-XCRUN="$(which xcrun)"
+XCRUN="$(which xcrun || true)"
```

Comment:
in this case, I believe that the value of `XCRUN` would be the path to the executable or the string `"true"`, right?
I don't think this is what we want. What we want is that:
* if `XCRUN` exists in the system, we want the path to it.
* otherwise, we want an empty string, not true.

with true, the next `if` will pass, because `if [ -n "$XCRUN" ]` is `true` but what we actually want is to enter the `else` branch, right?

Reply:
> What we want is that:
>  -  if XCRUN exists in the system, we want the path to it.
>  -  otherwise, we want an empty string, not true.

This is actually the case! `true` is a command that outputs nothing (so empty string). It's here in order to make the subshell invocation to return success (and prevent exiting) regardless of the result of calling `which xcrun`.

As opposed to this, which would have set it to `true` as fallback:
> `XCRUN="$(which xcrun || echo 'true')"`

Reply:
Ok, my bad... Sometimes bash eludes me!

Reply:
Thanks for review and approve!